src/pages/en/tutorial/3-components/2.md

```diff
@@ -163,38 +165,35 @@ import '../styles/global.css';
 
 <Box icon="question-mark">
 
-### Analyze the pattern
-
-1. What line of code do you need to write in an Astro component to **receive values** of `title`, `author`, and`date` **as props**?
-
-    || `const { title, author, date } = Astro.props;` ||
+### Test Yourself
+
+1. What line of code do you need to write in an Astro component's frontmatter to receive values of `title`, `author`, and `date` as props?
+
+    <MultipleChoice>
+      <Option isCorrect>
+        `const { title, author, date } = Astro.props;`
+      </Option>
+      <Option>
+        `import BlogPost from '../components/BlogPost.astro`
+      </Option>
+      <Option>
+        `<BlogPost title="My First Post" author="Dan" date="12 Aug 2022" />`
+      </Option>
+    </MultipleChoice>
     
 
 2. How do you **send values as props** to an Astro component?
```

Comment:
Is it too jargon-y to say “pass” here? I’m OK leaving this as-is if you’re using “send” in general on this page, but wonder if “pass” is a word we’re introducing with the concept.

```suggestion
2. How do you **pass values as props** to an Astro component?
```